### ISSUE TITLE
Module "mrcnn.utils" missing

### ISSUE THREAD

**Issue by wrznr** [mjenckel/ocrd_anybaseocr#32]
Please help! With current master, I receive:
```
$ ocrd-anybaseocr-block-segmentation -J
Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0120 13:29:43.066344 140353417594688 deprecation.py:323] From /home/kmw/Documents/Work/OCR-D/env/lib/python3.6/site-packages/tensorflow_core/python/compat/v2_compat.py:65: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term
Traceback (most recent call last):
  File "/home/kmw/Documents/Work/OCR-D/env/bin/ocrd-anybaseocr-block-segmentation", line 5, in <module>
    from ocrd_anybaseocr.cli.cli import ocrd_anybaseocr_block_segmentation
  File "/home/kmw/Documents/Work/OCR-D/env/lib/python3.6/site-packages/ocrd_anybaseocr/cli/cli.py", line 11, in <module>
    from ocrd_anybaseocr.cli.ocrd_anybaseocr_block_segmentation import OcrdAnybaseocrBlockSegmenter
  File "/home/kmw/Documents/Work/OCR-D/env/lib/python3.6/site-packages/ocrd_anybaseocr/cli/ocrd_anybaseocr_block_segmentation.py", line 30, in <module>
    from ocrd_anybaseocr.mrcnn import model
  File "/home/kmw/Documents/Work/OCR-D/env/lib/python3.6/site-packages/ocrd_anybaseocr/mrcnn/model.py", line 29, in <module>
    from ocrd_anybaseocr.mrcnn import utils
ImportError: cannot import name 'utils'
```
How can I get `utils`?

**Comment by wrznr** [comment]
The file was deleted in https://github.com/mjenckel/ocrd_anybaseocr/commit/0ac8567b1faffdd8bc5e1c67a74fb03761404661
How comes that it works for you?

**Comment by wrznr** [comment]
Consequently, `make test` fails.
```
$ make test
rm -rf tests/assets
mkdir -p repo/
git clone https://github.com/OCR-D/assets "repo/assets"
Cloning into 'repo/assets'...
remote: Enumerating objects: 549, done.
remote: Counting objects: 100% (549/549), done.
remote: Compressing objects: 100% (378/378), done.
remote: Total 3248 (delta 245), reused 443 (delta 158), pack-reused 2699
Receiving objects: 100% (3248/3248), 142.32 MiB | 14.27 MiB/s, done.
Resolving deltas: 100% (1172/1172), done.
mkdir -p tests/assets
cp -r -t tests/assets repo/assets/data/*
cd tests/assets/dfki-testdata/data && ocrd-anybaseocr-binarize -m mets.xml -I OCR-D-IMG -O OCR-D-IMG-BIN-TEST
Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0120 13:56:19.605401 139992437884736 deprecation.py:323] From /home/kmw/Documents/Work/OCR-D/env/lib/python3.6/site-packages/tensorflow_core/python/compat/v2_compat.py:65: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term
Traceback (most recent call last):
  File "/home/kmw/Documents/Work/OCR-D/env/bin/ocrd-anybaseocr-binarize", line 5, in <module>
    from ocrd_anybaseocr.cli.cli import ocrd_anybaseocr_binarize
  File "/home/kmw/Documents/Work/OCR-D/env/lib/python3.6/site-packages/ocrd_anybaseocr/cli/cli.py", line 11, in <module>
    from ocrd_anybaseocr.cli.ocrd_anybaseocr_block_segmentation import OcrdAnybaseocrBlockSegmenter
  File "/home/kmw/Documents/Work/OCR-D/env/lib/python3.6/site-packages/ocrd_anybaseocr/cli/ocrd_anybaseocr_block_segmentation.py", line 30, in <module>
    from ocrd_anybaseocr.mrcnn import model
  File "/home/kmw/Documents/Work/OCR-D/env/lib/python3.6/site-packages/ocrd_anybaseocr/mrcnn/model.py", line 29, in <module>
    from ocrd_anybaseocr.mrcnn import utils
ImportError: cannot import name 'utils'
Makefile:91: recipe for target 'test-binarize' failed
make: *** [test-binarize] Error 1
```

**Comment by mahmed1995** [comment]
Fixed c801b8c

**Comment by wrznr** [comment]
Okay, `utils` is back now and `make test` works for me. Thanks @mahmed1995
